### DISCUSSION TITLE
VMware uefi bios issues

### DISCUSSION THREAD

**Discussion by bkurt78** [ipxe/ipxe#1278] [Q&A] [answered]
Hello.  I'm working on getting some VMs to boot via ipxe and then I would like to use an embedded script to have them chainload from there.  When I download the default ipxe.iso image from boot.ipxe.org and use that to boot one of my VMs it seems to work with either UEFI or standard bios.  However, when I download the source and build a default ipxe.iso it won't boot off of that at all in EFI mode.  I also noticed the default ipxe.iso from boot.ipxe.org is much larger than the custom built one I created.  How can I create something like the one on boot.ipxe.org site with an embedded script?  Thank you.

**Comment by bkurt78** [comment, marked as answer]
My apologies.  Shortly after posting my question I believe I found the answer at the post below.  

https://github.com/ipxe/ipxe/discussions/614

**Comment by NiKiZe** [comment]
What did you try to search for to see if this question already had an answer? (So that we can improve on that search result)

Answered in https://github.com/ipxe/ipxe/discussions/1155#discussioncomment-8480915 and several other places 😉

**Reply by bkurt78** [reply]
I originally was doing some searches using vmware and uefi.  I believe a google search for ipxe default iso eventually lead me to the post that helped me out.  I was just struggling with the fact that the downloadable ipxe.iso was working, but the one I was generating was not.  That post mentioned that the main ipxe.iso on the boot.ipxe.org site has many options compiled in that the standard make file doesn't allow you to do.  That was very helpful to understand the difference.  Once I understood that, figuring out the rest wasn't too bad.  Hope that helps.
